packages/api/srcbook/index.mts

```diff
@@ -139,7 +144,8 @@ export async function createSrcbook(title: string, metadata: SrcbookMetadataType
 
 async function createSrcbookDir(basename: string = randomid()) {
   const dirname = Path.join(SRCBOOKS_DIR, basename);
-  await fs.mkdir(dirname, { recursive: true });
+  await fs.mkdir(dirname);
+  await fs.mkdir(Path.join(dirname, 'src'));
```

Comment:
Could combine both of these into `fs.mkdir(Path.join(dirname, 'src'), { recursive: true });`

Reply:
yeah I go back and forth on this since I find recursive version 'clever' in the sense it's implicitly doing two things that are both required, so instead I opted for explicit. I don't care about the perf here since this doesn't happen often is equally instant to a human in either case